src/main/java/com/lothrazar/cyclic/block/cable/energy/TileCableEnergy.java

```diff
@@ -27,6 +29,7 @@
   final CustomEnergyStorage energy = new CustomEnergyStorage(MAX, MAX);
   private final LazyOptional<IEnergyStorage> energyCap = LazyOptional.of(() -> energy);
   private final Map<Direction, Integer> mapIncomingEnergy = Maps.newHashMap();
+  private int energyLastSynced = -1; //fluid tanks have 'onchanged', energy caps do not
```

Comment:
ok i think i see how this works. when it syncs and sends a packet, this is energyLastSynced set locally, but not saved in the tiles nbt. but thats good because it just needs to check it a few ticks later, and then it can skip sending a packet completely if the cable is for example constantly full all the time.

Reply:
This can be changed to populate energyLastSynced with the energy value when reading in from the nbt. Wireless Energy needs some TLC but I am trying out some caching techniques for neighbouring tiles before updating.